### PR TITLE
Some DS file types

### PR DESCRIPTION
**Pull request by PlancakeJB**
Includes a list of some common DS file types and docs for the sound files (SDAT, SWAV, etc). These are just copies of existing documentation that I have rewritten to fix formatting. I have not added any game-specific files as I plan to do the rest of them at a later time.

### CODE REVIEW

File: docs/khds/common/file/type/sseq.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+| 0xE1 | BPM: 2 byte | TEMPO
+| 0xE3 | Value: 2 byte | SWEEP PITCH
+| 0xFF | NONE | EOT: End Of Track
+
+
+
+
+
+
+
```
Comment (Delta-473) [inline docs/khds/common/file/type/sseq.md:93]: 1 new line character behind line 86 is enough ;)

Comment (PlancakeJB) [reply docs/khds/common/file/type/sseq.md:93]: Whoops. I forgot to delete that cause I was rushing this

**Comment by PlancakeJB** [comment]
I removed those extra line breaks. It should be good now.
(P.S. there's so many commits because I'm editing from browser)

File: docs/khds/common/file-type.md [hunk 2]
```diff
@@ -1,4 +1,26 @@
...
+| arm9.bin | ARM9 executable code; can be either encrypted or decrypted
+| arm7.bin | ARM7 executable code
+| NFTR | Nitro Font; font data
+| NSBMD | Nitro System Binary Model; 3D model data. Often also holds textures and palettes (nsbtx)
+| NSBTX | Nitro System Binary Texture; texture and palette data
+| NSBCA | Nitro Character Animation; 3D model skeletal animations
+| NSBTP | Nitro Texture Pattern; texture-swapping animations
+| NSBTA | Nitro Texture Animation; texture UV-change animations
+| NSBMA | Nitro Material Animation; material-swap animations
+| [SDAT](file//type/sdat.md) | Sound Data; contains audio data in sub-files
```
Comment (Delta-473) [inline docs/khds/common/file-type.md:21]: idk if double slash // after file generate an issue for html doc. It's probably fine.

File: docs/khds/common/file/type/sbnk.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+* __Attack rate__ determines how fast the note reaches 100% amplitude. (See the first upward curve). Thus the highest value 127 means the sound reaches 100% amplitude in the shortest time; 0 means the longest time.
+
+* __Decay rate__ determines how fast the amplitude decays to 0% amplitude. Of course the sound will not drop to 0% but stops at sustain level. (See the first downward curve). Thus the highest value 127 means the sound reachs the sustain level in the shortest time; 0 means the longest time.
+
+* __Sustain level__ determines the amplitude at which the sound sustains. (See the horizonal part). Thus the highest value 127 means the sound sustains at 100% amplitude (no decay), while 0 means 0% (full decay).
+
+* __Release rate__ determines how fast the amplitude drops from 100% to 0%. Not from sustain level to 0%. (See the second downward curve). The value has the same meaning as Decay rate.
+
+See this [file](http://www.feshrine.net/hacking/doc/articulation.htm) for more details on how to interpret the articulation data. The __raw data__ column is the transformed value used for calculation.
```
Comment (Delta-473) [inline docs/khds/common/file/type/sbnk.md:150]: URL returns 404

**Review by Truthkey — APPROVED** [review]
Everything looks correct now!

**Pull request merged**